sdks/java/core/pom.xml

```diff
@@ -121,6 +121,13 @@
         </executions>
       </plugin>
 
+      <!-- Ensure that the Maven jar plugin runs before the Maven
+        shade plugin by listing the plugin higher within the file. -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
If we moved the overrides on the shade plugin to `pluginManagement` then would the ordering from the parent automatically fix all the submodules?